### ISSUE TITLE
'Settings' object has no attribute 'CHOICES_SEPARATOR'

### ISSUE THREAD

**Issue by contaval** [Pierre-Sassoulas/django-survey#41]
Hello guys!

I'd like to say first of all that I'm quite new to Django.

I created a Django Project with pyCharm and installed this package using "pipenv install --pre django-survey-and-report". Everything seems to go fine. I add the app to INSTALLED_APPS just like you say in your documentation but then, when I create a survey, I get the error:

Request Method: | POST
-- | --
http://127.0.0.1:8000/admin/survey/survey/add/
2.1.9
AttributeError
'Settings' object has no attribute 'CHOICES_SEPARATOR'
C:\Users\jgarcia\.virtualenvs\Surveys-8COo6NzA\lib\site-packages\django\conf\__init__.py in __getattr__, line 58
C:\Users\jgarcia\.virtualenvs\Surveys-8COo6NzA\Scripts\python.exe

Any idea of why this may be happening?

Thank you in advance!

**Comment by Pierre-Sassoulas** [comment]
Hello, thank you for using django-survey and for this bug report. This is probably due to the last change on the master branch that permit to choose the separator. The intent was to have a default value, but as evidenced by your experimental data, it does not work. You can fix this bug by adding the following in your `settings.py` : 

```python
CHOICES_SEPARATOR = "," # Or another separator, like | or ~
```

**Comment by contaval** [comment]
Hello Pierre! 

First of all, I'd like to thank you for your help. I was able to solve it by adding that code. I saw that you were doing some changes regarding to the CHOICES_SEPARATOR and I was not sure if it was a bug.

I was able to add a survey, however, when I click on "View on site" I get another error:
Request Method: | GET
http://127.0.0.1:8000/survey/2/
survey.views.survey_detail.SurveyDetail

And the same occurs when I try to add a "set of answers":

Request Method: | POST
Request URL: http://127.0.0.1:8000/admin/survey/response/add/
Django version: 2.1.9
Exception Type: IntegrityError
Exception Value: NOT NULL constraint failed: survey_response.survey_id

I suppose this is something that I did wrong during the installation. I made the "make migrations" and all that stuff, but as I said, I'm quite new with Django. 

Do you know which may be the problem?

**Comment by contaval** [comment]
ello Pierre!

First of all, I'd like to thank you for your help. I was able to solve it by adding that code. I saw that you were doing some changes regarding to the CHOICES_SEPARATOR and I was not sure if it was a bug.

Thank you again!

**Comment by Pierre-Sassoulas** [comment]
Closed in favour of #74
